nservicebus/hosting/nservicebus-host/index.md

```diff
@@ -25,6 +25,17 @@ If you want to avoid the scanning process you can explicitly configure the type
 snippet:ExplicitHostConfigType
 
 
+### Controlling assembly scanning using the command line
+
+A list of assemblies to scan can also be controlled using the `/scannedAssemblies` switch. If this option is used, the `NServiceBus.Host.exe` loads only assemblies that have been explicitly listed on the command line. Each assembly must be added using a separate switch:
+
+`NServiceBus.Host.exe /scannedAssemblies:"NServiceBus.Host" /scannedAssemblies:"MyMessages" /scannedAssemblies:"MyEndpoint"`
```

Comment:
Seems like it https://github.com/Particular/NServiceBus.Host/blob/develop/src/NServiceBus.Hosting.Windows/HostProgram.cs#L122

Didn't even know it was possible to pass the assemblies like that :)